frontend/src/components/fragments/user/user-emote-list.component.tsx

```diff
@@ -55,7 +79,11 @@ class EmoteList extends Component<any, any> {
   }
 
   @bind
-  private ownerIdFormatter(cell: string, row: any, column: any): JSX.Element {
+  private ownerIdFormatter(cell: number, row: any, column: any): JSX.Element {
+    if (!this.props.user) {
+      throw new Error('User must be signed in');
+    }
```

Comment:
This is one thing I don't love about typing.  Since we declared this field as optional, it makes us confirm it before we can move on.  However, we "know" it shouldn't be undefined.

We can fix this in the redux mapper below or by just "forcing" it to believe us that it's not undefined.